app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
Looking at that thread, it seems that that is to ensure HTML4 compatibility. There is actually already a message specified in the model. I think the reason we aren't seeing it is the way the controller handles the `create` action. Instead of checking the validations, we are instead going into a large conditional where we are triggering the first item and immediately failing as can be seen [here](https://github.com/18F/dolores-landingham-bot/blob/master/app/controllers/employees_controller.rb#L10) -- this instead presents a message about no such username existing in your slack organization. We could remove all of the HTML5 validations by reordering the conditional a bit. I will submit a commit with those changes and you can let me know which you prefer.